tasks-by-users/1khansaad/02-card-hover-project/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>card hover effect</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="cards">
+                <div class="face face1">
+                    <div class="contant">
+                        <img src="play store.png" width="100px">
+                        <h2>Play store</h2>
+                    </div>
+                </div>
+                <div class="face face2">
+                    <div class="contant">
+                        <p>Google Play, also branded as the Google Play Store and formerly Android Market, is a digital distribution service operated and developed by Google. It serves as the official app store for certified devices.</p>
+                        <a href="#">Read More</a>
+                    </div>
+                </div>
+            </div>
+            <div class="cards">
+                <div class="face face1">
+                    <div class="contant">
+                        <img src="gmail.png" width="100px">
```

Comment:
I would suggest you add the alt attribute to all the img elements...

Reply:
added alt atribute